KUBECON.md

```diff
@@ -0,0 +1,79 @@
+[![](https://lh3.googleusercontent.com/YyFFZKmhaF_tupq893gXB9QlI0sTIB_kKIWRQRKYkM4DM57NiWsA8-a3fdAsNfVSENMpmB8woL7ZoMxvz60QJ9E=w16383)Flux and Flagger @ KubeCon Paris 2024](/kubecon)
+
+# KubeCon activities!
+
+![](https://lh4.googleusercontent.com/Zs2nLOcdZduejxNPG1ggl7LC7LGW4e9ICzWj6PVeOvb3U4BA4VxylXIJbNIDUGB9K8lKVQrRhtJ8K7zc69cvVqggfFYepEqpA1c6-QQwZKfzzW6UT3P42uB9M6XpLk7eyA=w1280)
+
+# KubeCon Paris 2024
+
+March 19-22, 2024
+
+![](https://lh5.googleusercontent.com/QSY9peqmDRMHJ48d6C39-WFTsXZmWchXFWGo-sLsPqQjWiYlwgifgBTnpYmmQKmQxAmHcgvCSNvYX4Hy8OY21OHqd7S-05iWaULoskrqSdmspuH4lj31FFOBmIcwQH02kw=w1280)
```

Comment:
The image links don't seem to work, do we need to have those in website assets maybe?

Reply:
That will also make the content stable, since right now that link reference seems to change every time the Google site refreshes - it's some kind of dynamic cache key

Reply:
The google sites links are no longer in the output, they should be replaced by stable `figure` shortcodes which are an embedded shortcode in Hugo. 👍